rotkehlchen/db/ethtx.py

```diff
@@ -245,6 +245,17 @@ def get_transaction_hashes_no_receipt(
 
         return hashes
 
+    def get_all_transaction_hashes(self) -> List[EVMTxHash]:
+        cursor = self.db.conn.cursor()
+        cursor_result = cursor.execute('SELECT DISTINCT tx_hash FROM ethereum_transactions')
+        hashes = []
+        for entry in cursor_result:
+            try:
+                hashes.append(deserialize_evm_tx_hash(entry[0]))
```

Comment:
It's safe to do, but there is no need to iterate all transactions and pass them through a deserialization function. What's saved in the DB is already considered deserialized, and this would speed up the function. In fact there is no need for `get_all_transaction_hashes` to be a function at all. Can just be a cursor.execute but if you use it in many places can stay

Reply:
At the moment, it isn't used in any other place at all. I'll remove it then